app/experimenter/experiments/api/v5/types.py

```diff
@@ -157,3 +159,9 @@ def resolve_ready_for_review(self, info):
 
     def resolve_jexl_targeting_expression(self, info):
         return self.targeting
+
+    def resolve_is_enrollment_paused(self, info):
+        return self.is_paused
+
+    def resolve_enrollment_end_date(self, info):
+        return self.proposed_enrollment_end_date
```

Comment:
Oh interesting, do we not want the column to distinguish when enrollment actually ended (versus when it was expected to end), or is it impossible enrollment could end early? If they're always one and the same we could even do the start_date + proposed_enrollment math in the client (though this field is still handy).

Reply:
Yep very good point, I thought about that too.

We could look up the date it really ended by introspecting into the changelog to look for the date that `is_paused` flipped to True, but I think we should hold off on building out that logic until we have a strong case that the frequency of experiment enrollment end drifting from the proposed date is sufficient to warrant it.

Also yes in theory we could do that logic in the UI but I'd generally warn against doing that kind of logic on the frontend, particularly when there is an implementation of it on the backend.  We don't want to wind up in a situation where we have that logic duplicated, change it in one place, and then forget to do it somewhere else.